pkg/kubelet/container_bridge.go

```diff
@@ -43,15 +43,16 @@ func createCBR0(wantCIDR *net.IPNet) error {
 		glog.Error(err)
 		return err
 	}
-	// restart docker
+	// Stop docker so that babysitter process can restart it again with proper configurations and
+	// checkpoint file
 	// For now just log the error. The containerRuntime check will catch docker failures.
 	// TODO (dawnchen) figure out what we should do for rkt here.
 	if util.UsingSystemdInitSystem() {
-		if err := exec.Command("systemctl", "restart", "docker").Run(); err != nil {
+		if err := exec.Command("systemctl", "stop", "docker").Run(); err != nil {
```

Comment:
All other platforms we supported except building the cluster from the scratch which we don't know, they have their own babysitter processes, to monitor both kubelet and docker like what we do for gce / gke since 1.0 release. I cc-ed @justinsb But I could not change systemd node for this though. 

All your suggestion here is we have to duplicate logic in two places: one in babysitter, one in kubelet, which I don't want to. Babysitter process is part of core system already, and it is outside kubelet.